### PR TITLE
Creating story for `account-mismatch-warning`

### PR DESCRIPTION
**Pull request by dhruvv173**
## Explanation

- Created story for `account-mismatch-warning` component and replaced a deprecated constant
- #18714 
- `ui/components/ui/account-mismatch-warning`

#### Story does not exist in `ui/components/ui` folder
image

### After

image
image

covered `jest` tests
image

## Manual Testing Steps

- Pull this branch
- Run `yarn storybook`
- Check `AccountMismatchWarning` story in UI component

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by dhruvv173** [comment]
hey @georgewrmarshall could you please review this PR when possible?
Thanks!

File: ui/components/ui/account-mismatch-warning/account-mismatch-warning.stories.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import AccountMismatchWarning from './account-mismatch-warning.component';
+
+export default {
+  title: 'Components/UI/AccountMismatchWarning',
+
```
Comment (georgewrmarshall) [inline ui/components/ui/account-mismatch-warning/account-mismatch-warning.stories.js:6]: Nit: remove line break

**Review by georgewrmarshall — APPROVED** [review]
LGTM! Thanks for your contribution @dhruvv173

**Review by georgewrmarshall — DISMISSED** [review]
LGTM! Thanks for your contribution @dhruvv173

File: ui/components/ui/account-mismatch-warning/account-mismatch-warning.stories.js (hunk 1 above)
Comment (dhruvv173) [reply ui/components/ui/account-mismatch-warning/account-mismatch-warning.stories.js]: done!

**Review by georgewrmarshall — APPROVED** [review]

**Review by garrettbear — APPROVED** [review]

**Pull request merged**